### ISSUE TITLE
Accessing other pre-trained models in Table 1 and 2

### ISSUE THREAD

**Issue by yliu1240** [HICAI-ZJU/KANO#25]
Hi, all!
I'm wondering the other pre-trained models in Table 1 and 2 are models in their original papers or if you trained separate models using their strategies? Are these models publicly available as well (if you have trained separate versions)?

Thank you so much!

Best regards,

**Comment by Fangyinfff** [comment]
Hi, 

In our study, we compared KANO with 12 baseline methods, including GCN, GIN, N-GRAM, Hu et al., MGSSL, GraphMVP, MPNN, DMPNN, CMPNN, GEM, GROVER, and MolCLR. The results of GCN, GIN, N-GRAM, and Hu et al. were taken from the paper of MolCLR, while the results of MGSSL and GraphMVP were obtained from the original text of these two articles. We reproduced the results of MPNN, DMPNN, CMPNN, GEM, GROVER, and MolCLR based on their respective GitHub code repositories. To ensure a fair comparison, we followed the experimental setup of previous works, which involved conducting experiments using three different splits of train/val/test data.
